include/Engine/Bytecode/Compiler.h

```diff
@@ -19,6 +19,7 @@ class Compiler {
     static bool                 ShowWarnings;
     static bool                 WriteDebugInfo;
     static bool                 WriteSourceFilename;
+    static bool                 DoOptimizations;
```

Comment:
This should have a compiler setting to enable it (see `Compiler::Init()`)

Reply:
Fixed in f7b1ea018ca17e704572f2008e07d375859c3190.